src/components/Guilds/CreateProposalPage/ActionsBuilder/index.tsx

```diff
@@ -101,7 +105,11 @@ export const ActionsBuilder: React.FC<ActionsBuilderProps> = ({ editable }) => {
             key={idx}
             data={option}
             editable={editable && actionsEditMode}
+            showData={mintRep}
           />
+          {transferBuilder && editable && option.label === 'For' && (

```

Comment:
This isnt really how guild's competition style of voting is going to work as the against and all other options can have transfer actions also

(at least in the base implementation - we will support a required blank against later)